### PR TITLE
[Enhance]: Better support yolov3 pytorch2onnx

### PR DESCRIPTION
**Pull request by RunningLeon**
Better support yolov3 pytorch2onnx with TensorRT as backend. PR(#3956)

### CODE REVIEW

File: mmdet/models/dense_heads/yolo_head.py [hunk 1]
```diff
@@ -257,20 +257,24 @@ def _get_bboxes_single(self,
             # (h, w, num_anchors*num_attrib) -> (h*w*num_anchors, num_attrib)
             pred_map = pred_map.permute(1, 2, 0).reshape(-1, self.num_attrib)
 
-            pred_map[..., :2] = torch.sigmoid(pred_map[..., :2])
-            bbox_pred = self.bbox_coder.decode(multi_lvl_anchors[i],
-                                               pred_map[..., :4], stride)
+            xy = torch.sigmoid(pred_map[..., :2])
```
Comment (ZwwWayne) [inline mmdet/models/dense_heads/yolo_head.py:260]: We may add comments here to indicate why this change is necessary.

File: mmdet/models/dense_heads/yolo_head.py [hunk 2]
```diff
@@ -257,20 +257,24 @@ def _get_bboxes_single(self,
...
             cls_pred = torch.sigmoid(pred_map[..., 5:]).view(
                 -1, self.num_classes)  # Cls pred one-hot.
 
             # Filtering out all predictions with conf < conf_thr
             conf_thr = cfg.get('conf_thr', -1)
-            conf_inds = conf_pred.ge(conf_thr).nonzero().flatten()
-            bbox_pred = bbox_pred[conf_inds, :]
-            cls_pred = cls_pred[conf_inds, :]
-            conf_pred = conf_pred[conf_inds]
+            if conf_thr > 0:
```
Comment (ZwwWayne) [inline mmdet/models/dense_heads/yolo_head.py:272]: We may add comments here to indicate why this change is necessary. The previous code seems to be more general.

File: mmdet/models/dense_heads/yolo_head.py (hunk 1 above)
Comment (RunningLeon) [reply mmdet/models/dense_heads/yolo_head.py:260]: Pls. check here(#3956) for more detailed info.

File: mmdet/models/dense_heads/yolo_head.py (hunk 2 above)
Comment (RunningLeon) [reply mmdet/models/dense_heads/yolo_head.py:272]: Pls. check here(#3956) for more detailed info.

**Review by ZwwWayne — APPROVED** [review]
